README.md

```diff
@@ -63,8 +63,11 @@ export PATH=${PATH}:/Applications/Postgres.app/Contents/Versions/11/bin/
 
 To switch redis on you'll need to install it locally. On a OSX we've used brew for this. To use redis caching you need to switch it on by changing the config for development:
 
-        REDIS_ENABLED = True
+```
```

Comment:
Optional: For the paragraph above this, I'd suggest the following change for how to run redis locally:

You can run redis locally using

```
brew install redis
redis-server
```

I feel this is more explicit and easy to follow.

Reply:
Interesting. Happy to add `brew install redis` but I'd prefer to leave the reader to adopt their own approach e.g. brew prints out the following commands, which is what I use myself:

```
brew services start redis
brew services stop redis
```

OK if I just mention how to install it?

Reply:
Updated in https://github.com/alphagov/notifications-api/pull/3467/commits/6092baea643e5058237fd0d48e06f9d8f5918a65.

Reply:
Optional: I'd prefer we include explicit instructions in here for starting redis too. I would like our READMEs to be explicit where possible, and I think including a command for how to start redis would be useful. I don't mind whether that is your brew commands there or `redis-server` but it prevents either
- someone not realising they need to start a redis server
- having to immediately google, how to start redis

In particular for someone never having used redis before or someone not familiar with brew this may be useful.

So more of a paved path, very happy for people to deviate with their own preferences if they wish but ideally 90% of the time they should be able to follow the readme without needing to turn to google.

Reply:
Sure, I've added this in https://github.com/alphagov/notifications-api/pull/3467/commits/bd690ab71825b227b01d9fc3aa347dff146e1a5a.

An installation with Homebrew will recommend to use `services start`, so that's what I've gone with.